Sources/App/Errors/MoodError.swift

```diff
@@ -0,0 +1,46 @@
+import Vapor
+
+enum MoodError: Error {
```

Comment:
We'd need to find a different way or better name to deal with errors since this is specific to the project

Reply:
Maybe just `AppError`